bundle/jsky.app.ot/src/main/java/jsky/app/ot/viewer/action/EditItemTitleAction.java

```diff
@@ -41,6 +42,10 @@ public void actionPerformed(ActionEvent evt) {
 
     @Override
     public boolean computeEnabledState() throws Exception {
-        return isEditableContext() && getContextNode(ISPNode.class) != null;
+        if (isEditableContext()) {
+            ISPNode n = getContextNode(ISPNode.class);
```

Comment:
On the previous code there was a check for null on `n`. Why don't we need it anymore?